src/main/scala/cognite/spark/v1/RawTableRelation.scala

```diff
@@ -157,7 +199,7 @@ class RawTableRelation(
     }
 
     val rdd =
-      readRows(config.limitPerPartition, None, rawRowFilter, jsonSchema)
+      readRows(config.limitPerPartition, None, rawRowFilter, keysFilter, jsonSchema)
```

Comment:
`RawRowFilter` comes from the sdk, it's the filter that is passed to the list rows endpoint.... We could make it `Either[RawRowFilter, Seq[String]]`, but since `Either` usually means error or success, I'm not sure if it would lead to confusion

Reply:
Ah, I see, didn't realize it was in the SDK. Maybe call it something other than "filter" if it's not a `Filter`-like class, then? `requestedKeys`? They are not "required" as it's a valid query to ask for `key in ("foo", "bar")` even though "bar" doesn't exist (as is made clear by ignoring the 404 response above).